src/Controls/samples/Controls.Sample.SingleProject/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Windows Machine": {
       "commandName": "MsixPackage",
-      "nativeDebugging": false
+      "nativeDebugging": true
```

Comment:
Intentional?

Reply:
Yep. Just wanting to enable better exceptions... I hope.